packages/k8s/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor:
+# Maintainer:
```

Comment:
Could you please become the maintainer of this package?

Reply:
Yes, sure!

Reply:
Please could you put your name/email at line 2?